### PR TITLE
Js Capstone - Movie App

### PR DESCRIPTION
**Pull request by felixDev22**
**In this repo** 👍 

- [x] We created a new repo called Movies App 💪 
- [x] Used JavaScript to make websites dynamic and build basic single-page apps. 💯 
- [x] Used ES6 modules. ✅ 
- [x] Used callbacks and promises. 😄 
- [x] Used webpack.
- [x] Applied JavaScript best practices and language style guides in code.
- [x] Used AAA pattern for unit tests. ✅ 
- [x] Wrote units tests for a JavaScript app.
- [x] Followed Gitflow.
- [x] Solved simple git conflicts. 👍 😄 
- [x] Send and receive data from an API.
- [x] Use API documentation.
- [x] Perform a code review for a team member. 🎉

### CODE REVIEW

File: src/__tests__/functions.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+});
+
+describe.each([
+  [[10, 13, 14], 3],
+  [[10, 5, 2, 13, 14], 5],
+  [[10, 2, 13, 14, 2, 13, 14], 7],
+])('Counts the number of items displayed:', (a, b) => {
+  test('Should count items in an array', () => {
+    expect(mockItemCounter(a)).toBe(b);
+  });
```
Comment (franclobo) [inline src/__tests__/functions.test.js]: - **[OPTIONAL]** This is not explicit in the requirements but in order to have tests for all edge cases please 🙏🏽 add tests for all the counters (likes counter) and think in the case when there are no items on the page (0 items)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,172 @@
...
+# 📖 Javascript Capstone Project (Movie App) <a name="MovieMax"></a>
+
+> Js Capstone Project - This is a MovieMax app that sends and receives data from an API and populates data on the browser. It also allows you to add comments or likes to a movie. The app has implemented ES6 modules, callbacks, and promises, webpack, used AAA pattern for unit tests, and gitflow.
+
+## 👌 Kanban Template <a name="kanban">
+
+- Our Kanban template can be found [here.](https://github.com/felixDev22/Kanban---Movie-Project/projects/1)
+  
+## 🎉 Zoom Video
+- Our zoom video can be found [here.](https://drive.google.com/file/d/1E5l31oDKbgt5h0u7mVLxsu7HqJCZAkrv/view?usp=share_link)
```
Comment (franclobo) [inline README.md]: - [x] Kindly repeat the video to highlight some pieces of code that you are proud of. 👇🏽 

> You should also highlight some interesting piece of code or something you built that you are very proud of.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @felixDev22 and @leehaney254 , 

# ♻️ REQUIRED CHANGES ♻️

soClose

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- No linter errors ✔️
- Correct Git flow ✔️
- Descriptive PR ✔️
- Professional README file ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- When I opened your app it didn't show the list of movies until I refreshed the page, please 🙏🏽 correct this behavior for a better user experience.

- For a better user interface we could correct the number of columns of the items, there are just three in the mockup to avoid the horizontal scroll, also please 🙏🏽 the counter of the movies should be next to the `Movies` link instead of below and add a distance between the movie genre and the items 👇🏽

| Mockup | Your app |
|---|---|
| image | image |


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (leehaney254) [reply README.md]: **Thank you very much for the suggested changes. These are the changes we made:**

- [x] We adjusted the nav bar to enable the movie counter to be on one line
- [x] Changed the display columns to 3
- [x] Added test for 0 counter
- [x] Added distance between genre and the movie poster
- [x] Added function to allow fast image load when the window is loaded
- [x] Repeat the video recording

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @felixDev22 and @leehaney254,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
